src/gui/macutilities.mm

```diff
@@ -106,7 +106,14 @@ void openFiles(const PathList &pathList)
             for (const auto &path : pathList)
                 [pathURLs addObject:[NSURL fileURLWithPath:path.toString().toNSString()]];
 
-            [[NSWorkspace sharedWorkspace] activateFileViewerSelectingURLs:pathURLs];
+            // in some unknown way, the next line affects Qt's main loop causing the crash
```

Comment:
Could you reformat this comment so that every sentence starts with a capital letter?